jdbc/src/main/java/io/kestra/jdbc/runner/JdbcExecutor.java

```diff
@@ -190,6 +193,11 @@ public void run() {
     }
 
     private void executionQueue(Execution message) {
+        if (skipExecutionService.skipExecution(message.getId())) {
+            log.warn("Skipping execution {}", message.getId());
+            return;
+        }
+
```

Comment:
Move the filter on the executor side please to keep a track on the queue of that execution

Reply:
Can you be more explicit please? This code is inside the JdbcExecutor so I have no idea what you mean by "the executor side" and how to keep track on the queue of that execution.
I add it there as I wanted to avoid as much processing issue as possible so I thought it was best to filter as earlier as possible.

Reply:
- would have done this [here](https://github.com/kestra-io/kestra/blob/d102e2265d8874f3be00ec28cd0f7424be54df43/core/src/main/java/io/kestra/core/runners/ExecutorService.java#L58) so it will be done for every executor (whatever it's kafka or jdbc)